UnoCheck/RuntimePlatform/Core/IRuntimePlatform.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DotNetCheck.RuntimePlatform.Model;
+using System.Collections.Generic;
+
+namespace DotNetCheck.RuntimePlatform.Core
+{
+	public interface IRuntimePlatform
+	{
+		Platform Platform { get; }
+
+		IList<InstalledProgramInfo> GetInstalledPrograms();
+
+		bool IsInstalledProgram(string partialName);
+	}
+}
```

Comment:
I do not see the point of this interface (and the class RuntimePlatformBase) at this time. Let's remove it for now and we'll add it when we'll need to make generic code based on that.

Reply:
I removed!